core/offchain.go

```diff
@@ -235,10 +235,6 @@ func (bc *BlockChain) CommitOffChainData(
 					if !ok {
 						stats, err = bc.ReadValidatorStats(paid[i].Addr)
 						if err != nil {
-							utils.Logger().Info().Err(err).
```

Comment:
sorry, my bad. Not this line. It's this log "Could not roll up last continuous crosslink"